src/components/BulkUploaderFilePicker.tsx

```diff
@@ -0,0 +1,352 @@
+import React, {
+  ChangeEventHandler,
+  DragEventHandler,
+  MouseEventHandler,
+  useRef,
+  useState,
+} from 'react';
+import {
+  CheckCircleIcon,
+  DocumentPlusIcon,
+  XCircleIcon,
+} from '@heroicons/react/24/outline';
+import { DocumentArrowUpIcon } from '@heroicons/react/24/solid';
+import './BulkUploaderFilePicker.css';
+import { Spinner } from './Spinner';
+
+interface FilePickerMessageProps {
+  text: React.ReactNode;
+  icon?: React.ReactNode | null;
+  additionalText?: React.ReactNode | null;
+  resetLink?: boolean;
+}
+
+const FilePickerMessage = ({
+  text = '',
+  icon = null,
+  additionalText = null,
+  resetLink = false,
+}: FilePickerMessageProps) => (
+  <div className="file-picker-message">
+    {icon && (
+      <span className="file-picker-message__icon">
+        {icon}
+      </span>
+    )}
+    <span className="file-picker-message__text">
+      {text}
+    </span>
+    {additionalText && (
+      <span className="file-picker-message__additional-text">
+        {additionalText}
+      </span>
+    )}
+    {resetLink && (
+      <span className="file-picker-message__reset-link">
+        Reset uploader
+      </span>
+    )}
+  </div>
+);
+
+export const BulkUploaderFilePicker = () => {
+  const ACCEPTED_MIME_TYPES = ['text/csv'];
+  const ACCEPTED_FILE_EXTENSIONS = ['csv'];
+  const MAXIMUM_FILE_SIZE_BYTES = 1e+9; // 1GB
```

Comment:
Siding with the hard drive manufacturers, I see! :laughing:

Reply:
@jasonaowen 😆 Listen, I knew that value would change anyway…